modules/osd-aws-privatelink-architecture.adoc

```diff
@@ -16,7 +16,7 @@ The following diagram shows network connectivity of a PrivateLink cluster.
 
 .Multi-AZ AWS PrivateLink cluster deployed on private subnets
 
-image::156_OpenShift_ROSA_Arch_0621_privatelink.svg[Multi-AZ AWS PrivateLink cluster deployed on private subnets]
+image::156_OpenShift_ROSA_Arch_1221_privatelink.png[Multi-AZ AWS PrivateLink cluster deployed on private subnets]
```

Comment:
Just confirming: is the change from 0621 to 1221 intentional?

Reply:
Yup, they should still be the same image.